source/common/json/config_schemas.cc

```diff
@@ -128,11 +128,44 @@ const std::string Json::Schema::REDIS_PROXY_NETWORK_FILTER_SCHEMA(R"EOF(
 const std::string Json::Schema::TCP_PROXY_NETWORK_FILTER_SCHEMA(R"EOF(
   {
       "$schema": "http://json-schema.org/schema#",
-      "properties":{
+      "properties": {
         "stat_prefix" : {"type" : "string"},
-        "cluster" : {"type" : "string"}
+        "route_config" : {
+          "type" : "object",
+          "properties": {
+            "routes" : {
+              "type" : "array",
+              "items" : {
+                "cluster": {
+                  "type" : "string"
+                },
+                "source_ip_list" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                },
+                "source__ports": {
```

Comment:
thanks. Besides the typo there was an error in the schema. I fixed it and verified that without the fix the typo would have been caught by a test.